webapp/channels/src/i18n/fa.json

```diff
@@ -2684,7 +2684,6 @@
   "copied.message": "رونوشت شد",
   "copy.code.message": "رونوشت کد",
   "copy.text.message": "رونوشت متن",
-  "copy_url_context_menu.getChannelLink": "لینک را کپی کنید",
```

Comment:
i dont think we should be doing any addition/deletion in the other languages file. Can you revert these

Reply:
I wasn't sure about this, but I do know that we don't need these strings anymore. Are we sure these will be cleaned up in weblate later?

Reply:
I guess CI answered that for me :P